README.md

```diff
@@ -42,7 +63,7 @@ You can find listed here the API specification containing the opinionated model
 
 | Stream [version](https://github.com/Backbase/stream-services/releases) | Banking Services   | Java | Spring Boot |
 |------------------------------------------------------------------------|--------------------|------|-------------|
-| 4.0.0 to latest                                                        | 2023.12            | 17   | 3.1         |
```

Comment:
what about the version you just bumped here?

Reply:
done